portal-web/test/functional/com/liferay/portalweb/macros/asset/assetlist/AssetListsAdmin.macro

```diff
@@ -417,6 +417,15 @@ definition {
 		Alert.viewSuccessMessage();
 	}
 
+	macro gotoEllipsisMenuItemInItemSelector {
+		Click(
+			key_fieldLabel = "${fieldLabel}",
+			key_text = "ellipsis-v",
+			locator1 = "Categories#ANY_ITEM_SELECTOR");
+
+		MenuItem.click(menuItem = "${menuItem}");
```

Comment:
Yeah! This looks better than using the `AssetLists#PERSONALIZATION_DROPDOWN_ITEM` path.
As this macro has a specific function, you can pass the argument directly at macro layer, no need to be at test layer. If we were going to use this macro to click at another menu option, then passing an argument is more suitable.